lipsum/src/lipsum.c

```diff
@@ -144,12 +148,69 @@ plugin_init(G_GNUC_UNUSED GeanyData *data)
 	GtkWidget *menu_lipsum = NULL;
 	GKeyFile *config = g_key_file_new();
 	gchar *config_file = NULL;
+	gchar *config_file_old = NULL;
+	gchar *config_dir = NULL;
+	gchar *config_dir_old = NULL;
 	GeanyKeyGroup *key_group;
 
+
 	config_file = g_strconcat(geany->app->configdir,
 		G_DIR_SEPARATOR_S, "plugins", G_DIR_SEPARATOR_S,
 		"geanylipsum", G_DIR_SEPARATOR_S, "lipsum.conf", NULL);
 
+	#ifndef G_OS_WIN32
+	/* We try only to move if we are on not Windows platform */
+	config_dir_old = g_build_filename(geany->app->configdir,
+		"plugins", "geanylipsum", NULL);
+	config_file_old = g_build_filename(config_dir_old,
+		"lipsum.conf", NULL);
+	config_dir = g_build_filename(geany->app->configdir,
+		"plugins", "lipsum", NULL);
+	if (g_file_test(config_file_old, G_FILE_TEST_EXISTS))
+	{
+		if (dialogs_show_question(
+			_("Renamed plugin detected!\n"
+			  "\n"
+			  "GeanyLipsum has been renamed to Lipsum -- as you will "
+			  "already recognised it. \n"
```

Comment:
Could add another sentence to expand a bit like "As a result, the configuration old plugin configuration file will no longer be read".